Sample Data/Custom/SpyCloud_Watchlist_Data_CL.json

```diff
@@ -0,0 +1,82 @@
+[
+  {
+    "Document_Id": "a888d0f7-5688-471e-8230-8fd5ab903289",
+    "Domain": "example.net",
+    "Email": "vkonopelski@example.net",

```

Comment:
Please update the value for email, it must be sanitized@sanitized.com at all places.

Reply:
Updated